src/problem_bank_scripts/problem_bank_scripts.py

```diff
@@ -87,11 +87,9 @@ def parse_body_part(pnum, md_text):
 
     # Store the content of the level 2 header
     try:
-        content = codecs.unicode_escape_decode(
```

Comment:
I think there was a reason why I had done it this way initially...let me try to figure out why this was done this way, I don't want to break other questions by making this change.

Reply:
diff mode on GitHub said it was to support LateX, but that doesn't make too much sense to me, since the tests questions that use LaTeX seem to be fine with the changes here

Reply:
Thanks - so basically we are now allowing unicode characters in questions. Hopefully that fixes questions like this one: https://ca.prairielearn.com/pl/course/38/question/237974/preview